common/static/common/js/openCloseAccordion.js

```diff
@@ -0,0 +1,18 @@
+// TODO: adjust the pathname filter to account for all objects search/filter pages
+const initOpenCloseAccordionSection = () => {
+    document.addEventListener('DOMContentLoaded', function() {
+
+        let expandedSection = document.getElementById('accordion-open-close-section')
+        let pathname = window.location.pathname
+        if(expandedSection){
+            if (pathname.includes('search')){
+                expandedSection.classList.add('govuk-accordion__section--expanded');
+            } else if (!pathname.includes('search')){
+                expandedSection.classList.remove('govuk-accordion__section--expanded');
```

Comment:
This is a little pseudo cody but you could add something like this to the views for search and list
```
def get_form(self):
    form - super().get_form()
    
    nestest_div = form.helper.layout.find_by_id("accordion-open-close-section")
    current_class = nested_div.attrs.get("class", "")
    new_class = f"{current_class} govuk-accordion__section--expanded".strip()
    nested_dic.attrs["class"] = new_class
    return form
```
```suggestion
                expandedSection.classList.remove('govuk-accordion__section--expanded');
```